src/main/java/it/gov/pagopa/pu/organization/mapper/OrganizationMapper.java

```diff
@@ -45,4 +46,38 @@ public Organization toModel(OrganizationCreateDTO createDTO) {
     return organization;
   }
 
+  public OrganizationDetailDTO mapToDTO(Organization org) {
+    if (org == null) {
+      return null;
+    }
+
+    OrganizationDetailDTO dto = new OrganizationDetailDTO();
+    dto.setPassword(org.getPassword() != null ? encryptionService.decryptKey(org.getPassword()) : null);
+    dto.setGenerateNoticeApiKey(org.getGenerateNoticeApiKey() != null ? encryptionService.decryptKey(org.getGenerateNoticeApiKey()) : null);
```

Comment:
decruptKey handle null values, you could simplify avoiding to use the ternary operator